xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/nft/NfTokenOfferObject.java

```diff
@@ -0,0 +1,55 @@
+package org.xrpl.xrpl4j.model.client.nft;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.primitives.UnsignedInteger;
+import org.xrpl.xrpl4j.model.transactions.XrpCurrencyAmount;
+import org.xrpl.xrpl4j.model.transactions.Address;
+
+import java.util.Optional;
+
+public interface NfTokenOfferObject {
+
+  /**
+   * The amount of XRP, in drops, expected or offered for the token.
+   *
+   * @return the xrp currency amount
+   */
+  @JsonProperty("Amount")
+  XrpCurrencyAmount amount();
+
+  /**
+   * The intended recipient of the {@link org.xrpl.xrpl4j.model.client.accounts.NfTokenObject}.
+   * This address will receive the NFToken when the offer is accepted.
+   *
+   * @return The {@link Address} of the destination.
+   */
+  @JsonProperty("Destination")
+  Optional<Address> destination();
+
+  /**
+   * A hint indicating which page of the sender's owner directory links to this object, in case the directory
+   * consists of multiple pages.
+   * Note: The object does not contain a direct link to the owner directory containing it,
+   * since that value can be derived from the Account.
+   *
+   * @return A {@link String} containing the hint.
+   */
+  @JsonProperty("OwnerNode")
+  Optional<String> ownerNode();
```

Comment:
Is there an object for `OwnerNode` or is it just a `string` in XRPL?

Reply:
It's string and is for internal bookkeeping purposes.